<launch>
    <arg name="namespace" default="yk_builder"/>
    <arg name="num_trials" default="1"/>
    <arg name="start_num" default="40"/>
    <arg name="verbose" default="true"/>
    <arg name="connector_type" default="waterproof"/>
    <arg name="velocity_scale" default="0.02"/>
    <arg name="volume" default="75"/>
    <arg name="reset" default="false"/>
    <arg name="lift" default="false"/>
    <arg name="demo" default="false"/>
    <arg name="use_audio_terminator" default="true"/>
    <arg name="config" default="nist.yaml"/>
    
    <group ns="$(arg namespace)">
        <node pkg="vibro_tactile_toolbox" name="test_nist_audio_outcome_node" type="test_nist_audio_outcome.py" output="screen">
            <param name="namespace" value="$(arg namespace)"/>
            <param name="root_pwd" value="$(find vibro_tactile_toolbox)"/>
            <param name="num_trials" value="$(arg num_trials)"/>
            <param name="start_num" value="$(arg start_num)"/>
            <param name="connector_type" value="$(arg connector_type)"/>
            <param name="velocity_scale" value="$(arg velocity_scale)"/>
            <param name="volume" value="$(arg volume)"/>
            <param name="reset" value="$(arg reset)"/>
            <param name="lift" value="$(arg lift)"/>
            <param name="demo" value="$(arg demo)"/>
            <param name="use_audio_terminator" value="$(arg use_audio_terminator)"/>
            <param name="verbose" value="$(arg verbose)"/>
            <param name="config" value="$(arg config)"/>
        </node>
    </group>
</launch>